src/containers/token-bridge/components/ready-to-use.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
Where is this component used?

Reply:
Hmmm, it is not used anymore... I forgot to clean it up.